insights/specs/insights_archive.py

```diff
@@ -139,6 +139,7 @@ class InsightsArchiveSpecs(Specs):
     netstat_i = simple_file("insights_commands/netstat_-i")
     netstat_s = simple_file("insights_commands/netstat_-s")
     nmcli_dev_show = simple_file("insights_commands/nmcli_dev_show")
+    nmcli_conn_status = simple_file("insights_commands/nmcli_conn_show")
```

Comment:
Please move this spec ahead of the nmcli_dev_show for alphabet order.

Reply:
acknowledged, updated in the code in latest PR